pkg/mixin/testdata/remote-mixins/index.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "name": "az",
+    "author": "Porter Authors",
+    "description": "A mixin for using the az cli",
+    "sourceURL": "https://cdn.porter.sh/mixins/az",
```

Comment:
Again here in the test data, only source url or feed url should be set, not both.